### PR TITLE
[DPE-2731] Secret labels (6/edge)

### PR DESCRIPTION
**Pull request by juditnovak**
More details on original PR: https://github.com/canonical/mongodb-operator/pull/268

NOTE:
I've moved the LIBVERSION numbers to correspond to the `v0` directory where those libs reside -- as CI couldn't run otherwise.

To be rebased after https://github.com/canonical/mongodb-operator/pull/271 is merged -- until then downgrading `LIBVERSION` to have pipelines runnig

### CODE REVIEW

**Review by dmitry-ratushnyy — DISMISSED** [review]

**Comment by MiaAltieri** [comment]
Hi Judit, great observation on libs here. We already [have an open PR](https://github.com/canonical/mongodb-operator/pull/271) fixing this, hopefully we can get this merged asap it doesnt block your changes

**Review by MiaAltieri — DISMISSED** [review]
Thank you for your work here! Ideally we can wait until #271 is merged so those nasty lib issues go away

**Review by dmitry-ratushnyy — DISMISSED** [review]

**Review by delgod — DISMISSED** [review]

**Review by MiaAltieri — DISMISSED** [review]

**Review by dmitry-ratushnyy — APPROVED** [review]

**Review by delgod — APPROVED** [review]

**Pull request merged**
